docs/04-operation-guides/operations/05-upgrade-kyma.md

```diff
@@ -1,10 +1,18 @@
 ---
-title: Upgrade Kyma
+title: Upgrade or Downgrade Kyma
 ---
 
-You can use the `deploy` command not only to install Kyma but also to upgrade the Kyma version on the cluster. For example, you can specify the sources, components, and configuration values you want to use. For code examples, see [Install Kyma](02-install-kyma.md).
+## Upgrade
+
+You can use the `deploy` command not only to install Kyma, but also to upgrade the Kyma version on the cluster. For example, you can specify the sources, components, and configuration values you want to use.
+
+For code examples, see [Install Kyma](02-install-kyma.md).
 
 > **NOTE:** If you upgrade from one Kyma release to a newer one, the automatic compatibility check compares your current version and the new release.<br>
```

Comment:
Just FYI: This `</br>` tag only works in GitHub, it won't work for you on the Website, but it does no harm, so it's OK to leave it :) 

GitHub:
<img width="747" alt="image" src="https://user-images.githubusercontent.com/53601578/198278217-73e0a44b-2a8c-4eff-baac-33a73f527e78.png">
Website:
<img width="777" alt="image" src="https://user-images.githubusercontent.com/53601578/198278405-ceb4eca4-c0a4-44ff-be83-7b477620a678.png">